pavelib/assets.py

```diff
@@ -165,7 +165,7 @@ def collect_assets(systems, settings):
     `settings` is the Django settings module to use.
     """
     for sys in systems:
-        sh(django_cmd(sys, settings, "collectstatic --noinput > /dev/null"))
+        sh(django_cmd(sys, settings, "collectstatic --clear --noinput > /dev/null"))
```

Comment:
Would it be possible for you to fix update_assets to not do RequireJS Optimizer when in devstack/debug mode? The overhead is killing everybody who works with assets...